dwhdelft.nl/pages/actieplan.vue

```diff
@@ -0,0 +1,141 @@
+        - 'De gemeente is in gesprek gegaan met deelnemende verenigingen van het Sportakkoord om hun visie op LHBT+ op de website te publiceren als signaal dat deze groep welkom is. (Actie: gemeente Delft)'
+        - 'De gemeente gaat deelnemende partijen van de werkgroep Positief Sportklimaat (onderdeel Sportakkoord) voorleggen om LHBT+ een terugkerend onderwerp te maken op de maandelijkse agenda (Actie: gemeente Delft, Haaglanden Beweegt)'
+        - 'Op www.sportenindelft.nl wordt informatie gedeeld over genderneutrale/gemixte teams en competities (Actie: Delft voor Elkaar / Haaglanden Beweegt)'
+    - name: Welzijn
+      tasks:
+        - 'Gemeente Delft voert samen met het Klant Contact Centre een cliëntreis uit en op basis van de resultaten worden de werkwijze en protocollen opnieuw bekeken (Actie: gemeente Delft)'
+        - 'De psychosociale hulpverlening voor jongeren in Delft (bijv. het Jongeren Informatie Punt) heeft expertise op het gebied van LHBT+ en weet jongeren door te verwijzen naar lokale activiteiten en support netwerken (Actie: DWH)'
+        - 'Roze 50+ en gemeente Delft hebben gesprekken gevoerd met zorginstellingen om hun locaties LHBT+ vriendelijk te maken en hun beleid op de website te plaatsen (Actie: Roze 50+, gemeente Delft)'
+        - 'De Roze Salon als ontmoetingsplek voor LHBT+ ouderen in Delfshove wordt voortgezet.'
+    - name: Stad
+      tasks:
+        - 'De gemeente onderzoekt waar en hoe genderregistratie op gemeenteformulieren weggelaten kan worden (Actie: gemeente Delft)'
+        - 'De gemeente onderzoekt de mogelijkheden voor genderneutrale toiletten in algemene ruimtes van het gemeentekantoor (Actie: gemeente Delft)'
+        - 'Op Coming Out Day hijsen de wethouder en ambtenaar LHBT+ beleid samen met lokale inzetters voor LHBT+ emancipatie een pride vlag (Actie: gemeente Delft, DWH)'
+        - 'De gemeente behoudt en onderhoudt de LHBT+ symbolen in de stad, zoals de regenboogbrug en de elektriciteitskastjes (Actie: gemeente Delft)'
+        - 'De gemeente organiseert jaarlijks in de week voor Paarse Vrijdag een evaluatie met de betrokken partijen van het actieplan over de voortgang van het Regenboog Actieplan (Actie: gemeente Delft)'
+        - 'De gemeente werkt actief samen met andere regenbooggemeenten in de regio (Actie: gemeente Delft)'
+        - 'De gemeente maakt hun LHBT+ beleid en relevante voorzieningen voor de doelgroep zichtbaar op de website (Actie: gemeente Delft)'
+    - name: Wijken en geloofsgemeenschappen
+      tasks:
+        - 'Delftse Werkgroep Homoseksualiteit en de gemeente voeren gesprekken in wijken en bij diverse geloofsgemeenschappen over LHBT+ (Actie: DWH)'
+        - 'Delftse Werkgroep Homoseksualiteit werkt samen met organisaties in de stad aan LHBT+ zichtbaarheid en activiteiten in de wijken buiten het centrum (Actie: DWH)'
```

Comment:
```suggestion
        - 'DWH werkt samen met organisaties in de stad aan LHBT+ zichtbaarheid en activiteiten in de wijken buiten het centrum (Actie: DWH)'
```
Dat communiceren we niet meer

Reply:
Agree, maar dit is een copy van het document waar dat nog wel in staat